ScoutSuite/core/console.py

```diff
@@ -13,7 +13,7 @@
 
 mfa_serial_format = r'^arn:aws:iam::\d+:mfa/[a-zA-Z0-9\+=,.@_-]+$'
 re_mfa_serial_format = re.compile(mfa_serial_format)
-re_mfa_code = re.compile(r'^\d{6}\d*$')
+re_mfa_code = r'^\d{6}\d*$'
```

Comment:
```suggestion
re_mfa_code_pattern = r'^\d{6}\d*$'
```
I believe this would be a clearer variable name.

Reply:
Make sure to update the lines where it is used as well if you apply the suggestion 😉